DatadogSessionReplay/SRSnapshotTests/SRSnapshotTests/Utils/ImageComparison.swift

```diff
@@ -30,66 +30,116 @@ internal struct ImageLocation {
     }
 }
 
-/// Compares `newImage` against the snapshot saved in `snapshotLocation` OR updates stored snapshot image data (if `record == true`).
-internal func DDAssertSnapshotTest(
-    newImage: UIImage,
-    snapshotLocation: ImageLocation,
-    record: Bool,
-    file: StaticString = #filePath,
-    line: UInt = #line
-) {
-    DDAssertSimulatorDevice("iPhone14,7", "16.2", file: file, line: line)
+internal extension XCTestCase {
+    /// Compares `newImage` against the snapshot saved in `snapshotLocation` OR updates stored snapshot image data (if `record == true`).
+    func DDAssertSnapshotTest(
+        newImage: UIImage,
+        snapshotLocation: ImageLocation,
+        record: Bool,
+        file: StaticString = #filePath,
+        line: UInt = #line
+    ) {
```

Comment:
I don't see `self` being used so it would best to keep it as function rather than instance method, like any other `DDAssert*`. Using function allows to use the assertion outside of a `XCTestCase` subclass, the can be useful when combining assertions for example.